src/components/LineChart/story.utils.ts

```diff
@@ -0,0 +1,57 @@
+export const initData = [
```

Comment:
These are mocked data, not utils. Can we please move them to stories.mdx file?

Reply:
I use initData in LineChart.test.ts file as well, guess i should move them there as well?

Reply:
or you can move them in a file "mockedData.ts" 😃